### ISSUE TITLE
Rethink `build/tei` directory

### ISSUE THREAD

**Issue by humenda** [freedict/tools#20]
Generator scripts, including the teiaddphonetics script, write their output to
build/tei/ to not clutter the source directory with intermediate files. This
becomes a problem when using XML includes (and schemata, etc.). At the moment,
this needs manual intervention and it would be good to resolve this issue.  
A good example is the eng-pol dictionary.

**Comment by micha137** [comment]
I enabled XInclude processing in teiaddphonetics, which now generates a eng-pol-ponectics.tei that has XIncludes resolved. Isn't that enough?

**Comment by humenda** [comment]
I ran into several issues yesterday with `eng-pol` and a few similar
dictionaries, because after conversion, the dictionaries is in
`build/tei/xxx-yyy-phonetics.tei` and hence the XInclude paths are invalid
(referencing the dictionary source directory). When I read your code, I figured
that `xsltproc` has an option `--path` exactly for this purpose. For now, this
issue is resolved. Our quality assurance scripts will need tweaking similar to
`xsltproc`.
